MetaCocktailsSwiftData/Views/Search View/SearchCriteriaViewModel.swift

```diff
@@ -7,20 +7,38 @@
 
 import SwiftUI
 
+
 final class SearchCriteriaViewModel: ObservableObject {
 
     
     @Published var searchText: String = ""
-    @Published var cocktailComponents = Tags.createComponentArray().sorted(by: { $0.name < $1.name })
+    @Published var cocktailComponents = createComponentArray().sorted(by: { $0.name < $1.name })
     @Published var matchedCocktails = [MatchedCocktail]()
     @Published var preferredCount = 0
     @Published var sections = [ResultViewSectionData]()
     @Published var isLoading = true
-    @Published var spiritsCategories = Spirit.createSpiritCategoryArray().sorted(by: { $0.lowercased() < $1.lowercased() })
+    @Published var spiritsCategories = [Agave().type.rawValue, Brandy().type.rawValue, Gin().type.rawValue, Other().type.rawValue, Rum().type.rawValue, Vodka().type.rawValue, Whiskies().type.rawValue, Liqueur().type.rawValue, FortifiedWine().type.rawValue, Wine().type.rawValue, Bitters().type.rawValue, Amari().type.rawValue]
```

Comment:
Lets use the boozecategory enum here instead (also can you make it uppercase 'Boozecategory' that's my bad)